src/main/java/org/jabref/gui/groups/GroupNodeViewModel.java

```diff
@@ -166,7 +174,7 @@ public boolean equals(Object o) {
     @Override
     public String toString() {
         return "GroupNodeViewModel{" +
-                "displayName='" + displayName + '\'' +
+                "displayNameProperty='" + displayNameProperty + '\'' +
```

Comment:
_You should probably call here .getValue()

Reply:
fixed